docs/data/stashcache/vo-registration.md

```diff
@@ -0,0 +1,147 @@
+Managing VO Registration for the StashCache Federation
+======================================================
+
+This document describes the registration steps required to manage a VO's role
+in the StashCache Data Federation.
+The StashCache Federation allows a site or regional network to cache data frequently used on the OSG,
+reducing data transfer over the wide-area network and decreasing access latency
+for the VO's jobs.
+
+For general information about StashCache, see the [overview document](/data/stashcache/overview/).
+
+For general information about VO registration in the OSG Topology,
+see the [registration document](/common/registration/#registering-virtual-organizations).
+
+Site admins should work together with VO managers in order to perform these steps.
+
+
+Definitions
+-----------
+
+- **Namespace:** a directory tree in the federation that is used to find VO data.
+- **Public data:** data that can be read by anyone.
+- **Protected data:** data that requires authentication to read.
+
+
+Requirements
+------------
+
+In order for a VO to join the StashCache Federation, the following needs to be true:
+
+- The VO must already be registered in the OSG Topology.
+
+- The VO must pick one or more "namespaces" for their data.
+  A namespace is a directory tree in the federation where VO data is found.
+
+!!! note
+    Namespaces are global across the federation, so you must work with the StashCache Operations team
+    to ensure that your VO's namespaces do not collide with those of another VO.
+
+- The VO must allow one or more StashCache origins to host their data.
+  An origin will typically be hosted on a site owned by the VO.
+  For information about setting up an origin, see the [installation document](/data/stashcache/install-origin/).
+
+- The VO must allow one or more StashCache Caches to cache their data.
+  The more places a VO's data can be cached in, the bigger the data transfer benefit for the VO.
+  The majority of caches across OSG will automatically cache all "public" VO data.
+  Caching "protected" VO data will often be done on a site owned by the VO.
+  For information about setting up a cache, see the [installation document](/data/stashcache/install-cache/).
```

Comment:
I think the last 3 bullets are unnecessary here and we should indicate in each section below that unique namespaces, caches, and origins are required.

Reply:
Not sure exactly what you mean but I think my "Split out Requirements" commit should do it?